src/server/util/readingsUtils.js

```diff
@@ -277,6 +277,62 @@ const groupDatakWh = [
     }
 ];
 
+// These are the 2D arrays for units and conversions to feed into the database
```

Comment:
While it is generally a good idea to encapsulate reused code as you have done here, OED made the decision that all but kWh related units would not be in this util file and each test would need to define them. I'm sorry to ask this but can you please remove the Thing definitions from this file and remove it from the test files. Please let me know if you have questions/thoughts about this.

Reply:
No worries, thank you so much for reviewing the code and the heads up! We'll make the change as soon as possible.